music downloader/m.py

```
@@ -0,0 +1,49 @@
+
+        for index, video_url in enumerate(playlist.video_urls, start=1):
+            try:
+                yt = YouTube(video_url)
+                print(f"\n{index}. {yt.title}")
+
+                # Download best audio
+                stream = yt.streams.filter(only_audio=True).first()
+                out_file = stream.download(output_path=playlist_folder)
+
+                # Convert to mp3
+                base, ext = os.path.splitext(out_file)
```

Ruff (0.13.3)

Unpacked variable `ext` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)